src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/TrxFileHelper.cs

```diff
@@ -208,7 +208,7 @@ public string MakePathRelative(string path, string basePath)
             // and "\\dir1". We need to compare the first non-null token
             // to know if we've got differences.
             int i = 0;
-            for (i = 0; i < max && pathTokens[i].Length == 0 && basePathTokens[i].Length == 0; i++) ;
+            for (i = 0; i < max && pathTokens[i].Length == 0 && basePathTokens[i].Length == 0; i++)
```

Comment:
This change incorrectly incorporates the following `if` block into `for`. 

```suggestion
            for (i = 0; i < max && pathTokens[i].Length == 0 && basePathTokens[i].Length == 0; i++)
            {
            }
```

Reply:
Fixed